lang/py3/avro/datafile.py

```diff
@@ -82,7 +84,7 @@
 })
 
 # Codecs supported by container files:
-VALID_CODECS = frozenset(['null', 'deflate'])
+VALID_CODECS = frozenset(['null', 'bzip2', 'deflate', 'xz'])
```

Comment:
Can we move the strings to constants? So we can use them to construct the set, but also use them below. This will reduce the probability of having a typo, and also client applications can import these constants. WDYT?

Reply:
Strongly agreed! Just updated the PR :)